src/Http/Controllers/SwaggerController.php

```diff
@@ -2,9 +2,10 @@
 
 namespace L5Swagger\Http\Controllers;
 
-use File;
-use Request;
-use Response;
+use Illuminate\Support\Facades\File;
+use Illuminate\Support\Facades\Request;
```

Comment:
Use `\Illuminate\Http\Request` instead

Reply:
Not sure but we can't because we can't call statically `secure` on `\Illuminate\Http\Request`

Reply:
You are correct. sorry, let's leave it as it is for all cases